jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/DefaultNamespaceEnricherTest.java

```diff
@@ -77,59 +83,36 @@ public void createWithPropertiesAndUnknownTypeInKubernetesShouldAddNothing() {
     assertThat(klb.build().getItems()).isEmpty();
   }
 
-  @Test
-  public void createWithPropertiesInKubernetesShouldAddNamespace() {
+  @DisplayName("create namespace")
+  @ParameterizedTest(name = "with properties {0}")
+  @MethodSource("create_namespaceTestData")
+  void create_namespace(String description, String namespace, PlatformMode platformMode, Class<? extends HasMetadata> clazz,
+      String expectedName) {
```

Comment:
combining these tests makes it hard to understand what we're testing.

Most of them assert different things.

Reply:
Remove the parameterized test and try to add displaynames that describe what the test is doing, you'll probably see why it's not a good idea to combine them